spec/requests/api/v1/posters_requests_spec.rb

```diff
@@ -165,4 +165,31 @@
     expect(poster).to have_key(:img_url)
     expect(poster[:img_url]).to be_a(String)
   end
+
+  it "can create a poster" do
+    poster_params = {
+      name: "REGRET",
+      description: "Hard work rarely pays off.",
+      price: 89.0,
+      year: 2018,
+      vintage: true,
+      img_url: "https://plus.unsplash.com/premium_photo-1661293818249-fddbddf07a5d"
+    }
+
+    headers = {"CONTENT_TYPE" => "application/json"}
```

Comment:
Out of curiosity (because I am not familiar with this code), what does this line (179) do?

Reply:
It defines the CONTENT_TYPE header, which tells the server the format of the data being sent in the body of the request. For this its going to be shoveled into JSON data.

Reply:
Thank you so much, great explanation!